cmds/stage

```diff
@@ -172,14 +172,14 @@ stage_repository() {
     stage_sign_repository
 }
 
-# Usage: stage_iso <machine>
+# Usage: stage_iso
 # Copy images from the deployment directory (DEPLOY_DIR) of the installer
 # machine to the iso staging area (STAGING_DIR) that are specific to ISO image
 # generation.
 stage_iso() {
-    local machine="$1"
+    local machine="$(sed -ne 's|^\([^ ]\+\)\s*control.\+|\1|p' "${CONF_DIR}/images-manifest")"
```

Comment:
`local machine="$( awk '$2 == "control" { print $1 }' "${CONF_DIR}/images-manifest")"`

awk makes it a little simpler.  Your sed commands works though, so you can leave it as-is if you prefer.

Reply:
Simpler is better, I'll make the change.